vuepress/docs/next/tutorials/getting-started/openshift-install-by-operator.md

```diff
@@ -0,0 +1,152 @@
+---
+sidebarDepth: 2
+---
+# Installation on Red Hat OpenShift using the Operator
+
+## Overview
+The following tutorial shows how to install an Entando Application using the Entando Operator and covers a few common enterprise configurations. You may also find the alternative [manual instructions](./openshift-install.md) useful.
+
+Scenario 1 is similar to the Entando quickstart style of deployment which can be applied in many environments, including on developer laptops. The subsequent scenarios build on that initial setup but can also be performed independently. Unless otherwise noted, you have the freedom to keep or modify the default options when installing the Entando Operator and other resources in the tutorials. 
+
+## Prerequisites
+- OpenShift 4.8.x
+- Cluster-admin access to OpenShift for initial installation of the Entando Operator
+- Familiarity with the OpenShift console and operation
+
+## Add the Entando Operator to the OperatorHub
+A cluster admin can add the 7.0 version of the Entando Operator into the local OperatorHub using the following command.
+```shell
+oc apply -f https://raw.githubusercontent.com/entando/entando-releases/v7.0.0/dist/ge-1-1-6/samples/openshift-catalog-source.yaml
+```
+
+## Scenario 1 - Embedded Database
+The initial scenario deploys the operator and Entando Application in a single namespace. We'll start with the smallest application footprint by using an embedded database, although this is not recommended for production use cases.
+1. Locate the `Entando Operator` in the `Operators` → `OperatorHub`. Make sure to select the appropriate version.
+2. Click `Install` to view the `Entando Operator` install options
+3. Select `A specific namespace on the cluster` for the `Installation mode`
+4. Choose an empty namespace for `Installed Namespace`. You can create one from `Home` → `Projects` first, if needed, e.g. `entando-one`. 
+5. Click `Install` to install the operator into your target namespace
+6. The install may take a few minutes to complete after which you can click `View Operator` to see the operator in your namespace. You can also go to `Operators` → `Installed Operators` at any time and select it from there. 
+7. Now go to `EntandoApp` and click `Create EntandoApp`
+   - Keep the default `my-app` as your application name or select your own
+   - Select the EntandoApp version: `7.0`
+8. Provide an `Ingress Host Name` specific to your namespace, e.g. `my-app.YOUR-BASE-OPENSHIFT-URL`. In CRC you can keep the default `entando.apps-crc.testing` for your first project.
+9. Change the `DBMS` to `embedded`. This is the lightest and quickest way to test a full Entando Application. However, a non-embedded relational database is strongly recommended for production use.
```

Comment:
embedded should not be backticked (right?? was not above and is a field value option)

DBMS also not backticked above